apisix/plugins/google-cloud-logging.lua

```diff
@@ -136,23 +136,23 @@ local function send_to_google(oauth, entries)
 end
 
 
-local function get_auth_config(config)
+local function get_auth_config(conf)
     if auth_config_cache then
         return auth_config_cache
     end
 
-    if config.auth_config then
-        auth_config_cache = config.auth_config
+    if conf.auth_config then
+        auth_config_cache = conf.auth_config
         return auth_config_cache
     end
 
-    if not config.auth_file then
+    if not conf.auth_file then
         return nil, "configuration is not defined"
     end
```

Comment:
why not use if ... else

Reply:
`conf.auth_config` and `conf.auth_file` is not  the same attribute

Reply:
sorry……